StarkInfra/StarkInfra/CreditNote/Invoice/Invoice.cs

```diff
@@ -16,14 +16,14 @@ namespace StarkInfra
     ///     <item>Amount [integer]: Invoice value in cents. Minimum = 1 (any value will be accepted). ex: 1234 (= R$ 12.34)</item>
     ///     <item>Due [DateTime or string, default now + 2 days]: Invoice due date in UTC ISO format. ex: "2020-10-28T17:59:26.249976+00:00" for immediate invoices and "2020-10-28" for scheduled invoices</item>
     ///     <item>Expiration [integer, default 5097600 (59 days)]: time interval in seconds between due date and expiration date. ex: 123456789</item>
-    ///     <item>Fine [float, default 2.0]: Invoice fine for overdue payment in %. ex: 2.5</item>
-    ///     <item>Interest [float, default 1.0]: Invoice monthly interest for overdue payment in %. ex: 5.2</item>
     ///     <item>Tags [list of strings, default null]: list of strings for tagging</item>
     ///     <item>Descriptions [list of Description objects, default null]: list of Description objects or dictionaries with "key":string and (optional) "value":string pairs</item>
```

Comment:
Não seria `[list of CreditNote.Invoice.Description objects]` ?

A docstring no SDK de Python está assim:
`descriptions [list of creditnote.invoice.Description objects or dictionaries, default None]: list Description objects`

Achei meio estranho como ficou em Python, quando chegar nesse ponto me chama pra gente definir junto.